storybook/qmlTests/tests/tst_SwapModal.qml

```diff
@@ -47,7 +47,7 @@ Item {
         signal fetchSuggestedRoutesCalled()
     }
 
-    readonly property var swapAdaptor: SwapModalAdaptor {
```

Comment:
why?

Reply:
just a sugar, syntax highlighting & code follow in Qt Creator